examples/getting_started/tokenizer_t5/config.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Maybe move this outside of getting_started?

Reply:
fixed